components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -214,31 +213,35 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                 )}
                 {showSpinner && (
                     <div className="flex flex-col mt-4 h-52 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
-                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance Used</div>
+                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance</div>
                         <Spinner className="m-2 h-5 w-5 animate-spin" />
                     </div>
                 )}
                 {showBalance && (
                     <div className="flex flex-col mt-4 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
-                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance Used</div>
+                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Balance</div>
                         <div className="mt-1 text-xl font-semibold flex-grow">
-                            <span className="text-gray-900 dark:text-gray-100">{used}</span>
-                            <span className="text-gray-400 dark:text-gray-500"> / {totalAvailable} Credits</span>
+                            <span className="text-gray-900 dark:text-gray-100">{currentUsage * -1 + usageLimit}</span>
```

Comment:
**question**: What do you think of using the same color for the balance and label (Credits) or dropping the label altogether? See https://github.com/gitpod-io/gitpod/pull/14848#discussion_r1029255558.